lib/foreman_maintain/config.rb

```diff
@@ -25,6 +25,11 @@ def initialize(options)
       @foreman_port = @options.fetch(:foreman_port, 443)
     end
 
+    def use_color?
+      # TODO: read config option?
+      ::ENV['TERM'] && !`which tput 2> /dev/null`.empty? && `tput colors`.to_i > 0
```

Comment:
`command -v` instead of `which` please

Reply:
Then we should update kafo too because I copied it from there

Reply:
Quite possibly, yeah.

Reply:
Now that I'm not on mobile I can easily find it: https://github.com/theforeman/kafo/blob/1e0258a8cc643b2bbef06eb0935d6ddf7681b87b/lib/kafo/color_scheme.rb#L6-L8